testing/dart/platform_dispatcher_test.dart

```diff
@@ -2,8 +2,13 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+import 'dart:io';
```

Comment:
I'm not entirely sure if this test is expected to run in a web browser, but importing dart:io probably will break that?

Reply:
Hmmm. I don't _think_ this test is intended to run in the web. `web_ui` has it's own set of tests to run as far as I know, and this test should be extending `dart:ui` ones.